warehouse/organizations/tasks.py

```diff
@@ -42,6 +42,20 @@ def update_organization_invitation_status(request):
         try:
             token_service.loads(invite.token)
         except TokenExpired:
+            invite.user.record_event(
+                tag=EventTag.Account.OrganizationRoleExpireInvite,
+                ip_address=request.remote_addr,
+                additional={
+                    "organization_name": invite.organization.name,
+                },
+            )
+            invite.organization.record_event(
+                tag=EventTag.Organization.OrganizationRoleExpireInvite,
+                ip_address=request.remote_addr,
+                additional={
+                    "target_user_id": str(invite.user.id),
+                },
+            )
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏coverage reports that these lines are covered, likely due to the existence of another test that exercises the `TokenExpired` behavior, but I didn't see any updates to testing the tasks that these events are recorded - are they tested some other way?

Reply:
Added asserts in 5a0ed16dc83b8f8b77d146cbba0e6f59c1e5b429